job-service/src/main/java/com/hpe/caf/services/job/api/JobsGet.java

```diff
@@ -57,19 +57,30 @@
 
             final JobSortField sortField;
             final SortDirection sortDirection;
+            final String sortByLabelValue;
```

Comment:
I've thought about it further and think it would be worthwhile to introduce that inheritance mechanism that we talked about, to correctly eliminate the either/or situation with `sortField` and `sortByLabelValue`.  It would only be one additional interface and one additional class after all, so go ahead and update the code to take that approach instead.

Reply:
implemented